src/app/ui/AppProvider/ToastProvider.tsx

```diff
@@ -13,8 +23,10 @@ const ToastButton = styled(Box, {
   padding: 10,
 });
 
-type ToastProps = {
+export type ToastProps = {
   message: string | JSX.Element;
+  icon?: React.ReactNode;
+  type?: string;
```

Comment:
I'm not sure a freeform `string` type is appropriate here. I would type it as below:

```suggestion
  type?: 'success' | 'warning' | 'danger' | 'info';
```

since the Banner component is prescriptively styled in Figma. This also improve developer experience as you can just rely on built-in autocomplete rather than have to open you component code to figure out what options are allowed.

(It's also fine if you want to limit your implementation to just `success` and `danger` for the time being, since that's all you need for your feature, but I'd still be prescriptive in your typing.)

Reply:
Also adding `warning` and `info` impl ;)